webapp/packages/core-utils/src/__custom__mocks__/ErrorBoundaryMock.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+type Props = {
+  children: React.ReactNode;
+};
+
+type State = {
+  hasError: boolean;
+  error: any | null;
+};
+export default class ErrorBoundary extends React.Component<Props, State> {
+  constructor(props: any) {
+    super(props);
+    this.state = { hasError: false, error: null };
+  }
+
+  static getDerivedStateFromError(error: any) {
+    return { hasError: true, error };
+  }
+
+  render() {
+    if (this.state.hasError) {
+      return <h1>{this.state.error.message}</h1>;
+    }
+
+    return this.props.children;
+  }
+}
```

Comment:
will be better to move it to `core-blocks` cuz we have `ErrorBoundary` component here